### PR TITLE
[ocean] Remove ES 2.x mappings to index Git data

### PR DESCRIPTION
**Pull request by valeriocos**
This patch removes mappings for ES 2.x

### CODE REVIEW

**Comment by acs** [comment]
Why are you removing the mappings for ES 2.x? We must support this version yet.

**Comment by valeriocos** [comment]
Most Ocean backends don't provide this feature. I thought that those mappings were dead code

**Comment by acs** [comment]
This removal can cause regressions so I won't do that.

File: grimoire_elk/ocean/git.py [hunk 1]
```diff
@@ -39,38 +39,21 @@ def get_elastic_mappings(es_major):
         :returns:        dictionary with a key, 'items', with the mapping
         """
 
-        if es_major == '2':
```
Comment (acs) [inline grimoire_elk/ocean/git.py]: This removal can cause regressions so I won't remove the 2.x support.

File: grimoire_elk/elk/git.py [hunk 2]
```diff
@@ -75,8 +75,8 @@ def get_elastic_mappings(es_major):
             {
                 "properties": {
                    "message_analyzed": {
-                      "type": "string",
-                      "index": "analyzed"
+                        "type": "string",
+                        "index": "analyzed"
```
Comment (acs) [inline grimoire_elk/elk/git.py]: Some strange spaces here ... could you review it?

**Review by acs — COMMENTED** [review]
LGTM

**Review by acs — APPROVED** [review]
LGTM

**Pull request merged**